index.html

```diff
@@ -2,71 +2,268 @@
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card__lango flex">
+            <li class="card__lists">html</li>
+            <li class="card__lists">css</li>
+            <li class="card__lists">javaScript</li>
+          </ul>
+          <button class="card__button" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card flex flx-dir-col">
+        <img
+          class="card__img"
+          src="./images/snapshoot/snapshot_3.png"
+          alt=""
+
+        />
+        <div class="card__hero flex flx-dir-col">
+          <h4 class="card__heading">Tonic</h4>
+          <ul class="card__tags flex">
+            <li class="card__items--dark">CANOPY</li>
+            <li class="dot"></li>
+            <li class="card__items">Back End Dev</li>
+            <li class="dot"></li>
+            <li class="card__items">2015</li>
+          </ul>
+          <p class="card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card__lango flex">
+            <li class="card__lists">html</li>
+            <li class="card__lists">css</li>
+            <li class="card__lists">javaScript</li>
+          </ul>
+          <button class="card__button" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card flex flx-dir-col">
+        <img
+          class="card__img"
+          src="./images/snapshoot/snapshot_4.png"
+          alt=""
+         
+        />
+        <div class="card__hero flex flx-dir-col">
+          <h4 class="card__heading">Multi-Post Stories</h4>
+          <ul class="card__tags flex">
+            <li class="card__items--dark">CANOPY</li>
+            <li class="dot"></li>
+            <li class="card__items">Back End Dev</li>
+            <li class="dot"></li>
+            <li class="card__items">2015</li>
+          </ul>
+          <p class="card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card__lango flex">
+            <li class="card__lists">html</li>
+            <li class="card__lists">css</li>
+            <li class="card__lists">javaScript</li>
+          </ul>
+          <button class="card__button" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="info__items flex flx-dir-col">
+        <h1 class="info__heading">About Myself</h1>
```

Comment:
- [ ] I would suggest you please use the `hx` tag like h2,h3,h4, and h5 as there is only one `h1` tag for pages and you have used it in `I’m Isa Glad to see you!` because each page should have only one main topic and headline. And why we use only h1 tag at once.